internal/rancher/payload_test.go

```diff
@@ -34,11 +34,13 @@ const payloadKubeconfigToken = `{
 }`
 
 const payloadKubeconfigTokenCached = `{
+  "created": "2021-03-25T10:38:18Z",
```

Comment:
Setting the `created` value to something in the past causes the cached token tests to fail, so something is off with the logic.

Reply:
Updated. Had to change one of the "shortExpiration" values being set for one of the tests. That change makes sense since the seconds check for the `createdAt` will now be the large value.